run_route_scripts/run_with_server.py

```diff
@@ -120,3 +121,17 @@ def make_request(post_body):
         progress = int(next_progress / increment)
     if progress != 100 / increment:
       print('100%')
+
+  # print total duration
+  if parsed_args.format == "json":
+    output_dir_path = Path(parsed_args.output_dir)
+    duration = 0
+    for out_f in output_dir_path.iterdir():
+      if out_f.is_file() and out_f.suffix == ".json":
+        with open(out_f) as f:
+          try:
+              duration += json.load(f)["performance"]["response_time"]
+          except json.decoder.JSONDecodeError:
+              continue
+
+    print(f"Requests took {duration} seconds")
```

Comment:
accumulate all durations of all requests so we can see how long each run took

Reply:
I always run this with `time` but this is good